spec/index.bs

```diff
@@ -4020,6 +4020,14 @@ GPUCommandBuffer includes GPUObjectBase;
         </div>
 </dl>
 
+{{GPUCommandBuffer}} has the following internal slots:
+
+<dl dfn-type=attribute dfn-for="GPUCommandBuffer">
+    : <dfn>\[[command_list]]</dfn> of type `sequence<algorithm>`.
```

Comment:
I think defining a command type would be good; it could have a nice explicit name and short definition like `command executor` or something.

nit: I'd probably call it a list or array since sequence is an IDL thing that takes iterables (I think). Not sure what the prior art does.

Reply:
Prior art that I'm aware of uses [list](https://infra.spec.whatwg.org/#list), which has a bunch of helpful algorithms already defined for it and I found very useful in WebXR. Thanks for reminding me that it's a thing. :)

Reply:
I had forgotten that doc was a thing (have only seen it, a while ago, while looking for some other prior art, I think). I should definitely read through it.